test/functional/rpc_scanblocks.py

```diff
@@ -94,6 +98,13 @@ def run_test(self):
         assert(genesis_blockhash in node.scanblocks(
             "start", [{"desc": f"raw({false_positive_spk.hex()})"}], 0, 0)['relevant_blocks'])
 
+        # check that the filter_false_positives option works
+        assert(genesis_blockhash in node.scanblocks(
+            "start", [{"desc": f"raw({genesis_coinbase_spk.hex()})"}], 0, 0, "basic", True)['relevant_blocks'])
+        assert(genesis_blockhash not in node.scanblocks(
+            "start", [{"desc": f"raw({false_positive_spk.hex()})"}], 0, 0, "basic", True)['relevant_blocks'])
+
+
         # TODO: after an "accurate" mode for scanblocks is implemented (e.g. PR #26325)
         # check here that it filters out the false-positive
```

Comment:
These comment lines from the previous commit can be removed now.

Reply:
Removed thanks